src/content/reference/faqs.mdx

```diff
@@ -246,3 +246,15 @@ You can only create `Roles` restricted to the endpoints above. `RoleBindings` ca
 This is related to how syncthing interacts with `tsc`. Syncthing creates a temporary file and replaces the original file with the new one.
 
 To solve the problem you just add the flag `--synchronousWatchDirectory` to your `tsc` command.
+
+## I cannot connect to the Kubernetes cluster using the kubeconfig file generated by Okteto
+
+Since CLI version `2.20`, the Kubeconfig file generated by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to get the credentials for your Kubernetes clusters from your Okteto instance. It is a common pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters in a more secure way.
```

Comment:
```suggestion
Since Okteto CLI version `2.20`, the Kubeconfig file generated by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to get the credentials for your Kubernetes clusters from your Okteto instance. It is a typical pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters in a more secure way.
```